src/ufixed.jl

```diff
@@ -31,29 +33,40 @@ const uf14 = UFixedConstructor{UInt16,14}()
 const uf16 = UFixedConstructor{UInt16,16}()
 
 zero{T,f}(::Type{UFixed{T,f}}) = UFixed{T,f}(zero(T),0)
-@generated function one{T<:UFixed}(::Type{T})
-    f = 2^nbitsfrac(T)-1
-    :( T($f,0) )
+function one{T<:UFixed}(::Type{T})
+    T(typemax(rawtype(T)) >> (8*sizeof(T)-nbitsfrac(T)), 0)
```

Comment:
the generated version might be faster.  see https://github.com/JuliaMath/FixedPointNumbers.jl/pull/44#r74753925.  or does the use of `>>` instead of `^` close the gap?